opba-protocols/xs2a-protocol/src/main/java/de/adorsys/opba/protocol/xs2a/service/xs2a/consent/ReportConsentAuthorizationRevoked.java

```diff
@@ -0,0 +1,26 @@
+package de.adorsys.opba.protocol.xs2a.service.xs2a.consent;
+
+import de.adorsys.opba.protocol.xs2a.domain.dto.messages.ConsentAcquired;
+import de.adorsys.opba.protocol.xs2a.service.ValidatedExecution;
+import de.adorsys.opba.protocol.xs2a.service.xs2a.RedirectExecutor;
+import de.adorsys.opba.protocol.xs2a.service.xs2a.context.Xs2aContext;
+import lombok.RequiredArgsConstructor;
+import org.flowable.engine.delegate.DelegateExecution;
+import org.springframework.stereotype.Service;
+
+@Service("xs2aReportToFintechConsentRevoked")
+@RequiredArgsConstructor
+public class ReportConsentAuthorizationRevoked extends ValidatedExecution<Xs2aContext> {
```

Comment:
ReportConsentAuthorizationDenied - it is not revoked as no consent actually exists

Reply:
done